packages/blocks/src/list-block/list-block.ts

```diff
@@ -190,9 +191,6 @@ export class ListBlockComponent extends BlockElement<
 
     return html`
       <div class=${'affine-list-block-container'}>
-        <style>
```

Comment:
This may cause the loss of styles in the AI text renderer, and we may need to disable the shadow DOM involving the AI text renderer. cc @donteatfriedrice

Reply:
I will check again.